test-two/src/services/github.js

```diff
@@ -6,11 +6,13 @@ const getAuthenticationQueryString = async apiAuthenticationToken =>
 export const getUsersForOrganisation = async (
   baseUrl,
   apiAuthenticationToken,
-  organisationId
+  organisationId,
+  max_per_page,
+  page
 ) => {
   return get(
-    `${baseUrl}orgs/${organisationId}/members${await getAuthenticationQueryString(
+    `${baseUrl}orgs/${organisationId}/members?&${await getAuthenticationQueryString(
```

Comment:
is there a reason for `?&` instead of just `?` here?

Reply:
No reason, I think deleting it will not make any difference in the code, I had it written this way 

`${baseUrl}orgs/${organisationId}/members?page=${page}&per_page=${max_per_page}&${await getAuthenticationQueryString(
  apiAuthenticationToken
)}&`
and swapped later but did not remove the "?&"